src/Pages/Loading.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { LoadingPage } from "../Components";
+
+const NotFound: React.FC = () => {
+  return <LoadingPage />;
+};
+
+export default NotFound;
```

Comment:
```suggestion
const Loading: React.FC = () => {
  return <LoadingPage />;
};

export default Loading;
```
이런식으로 네이밍을 수정해야 할 듯 해요!